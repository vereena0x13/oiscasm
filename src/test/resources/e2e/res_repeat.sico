            0 0 start

t0:         .res 1
xs:         .res 4, 7

start:      t0 t0 start

.repeat 4 ?+i

.repeat 4, x 1 << x

.repeat 4, y {
    y
    1 << y
    y ** 2
}