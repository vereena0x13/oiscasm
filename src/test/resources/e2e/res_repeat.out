0
0
8
0
7
7
7
7
3
3
8
11
13
15
17
1
2
4
8
0
1
0
1
2
1
2
4
4
3
8
9